bin/otomi

```diff
@@ -48,6 +48,7 @@ cmd_image="otomi/core:$(otomi_image_tag)"
 docker_terminal_params='-it'
 helm_config=''
 readme_url='https://github.com/redkubes/otomi-core'
+kubeconfig="-v ${KUBECONFIG:-$HOME/.kube/config}:/home/app/.kube/config"
```

Comment:
helm does it ;)

Reply:
The reason this var is defined as is is because it uses a removed line, see this commit:
https://github.com/redkubes/otomi-core/commit/973a5305131f6fc2a99c29e7af07c2bb67593324

Reply:
Is there NO test in place to detect this?

Reply:
A possible place is to detect it to create PR ;)

Reply:
@Morriz you added a `$kubeconfig` variable, without defining it anywhere?